AspNetCore3ChildlifetimeScope/StartupApplicationA.cs

```diff
@@ -0,0 +1,33 @@
+using AspNetCore3ChildlifetimeScope.Dependencies;
+using Autofac;
+using Autofac.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace AspNetCore3ChildlifetimeScope
+{
+    public class StartupApplicationA
+    {
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+        }
+
+        public void Configure(IApplicationBuilder app)
+        {
+            app.UseRouting();
+
+            app.UseEndpoints(builder => builder.MapControllers());
+        }
+
+        public void ConfigureContainer(AutofacChildLifetimeScopeConfigurationAdapter configurationAdapter)
```

Comment:
Some sort of comment here (and in the associated `StartupApplicationB` class) would help clarify this as a requirement.
```suggestion
        // You must use an AutofacChildLifetimeScopeConfigurationAdapter
        // here instead of a ContainerBuilder.
        public void ConfigureContainer(AutofacChildLifetimeScopeConfigurationAdapter configurationAdapter)
```